python/MetrologyData.py

```diff
@@ -342,29 +274,6 @@ def plot_statistics(self, nsigma=5, title=None, zoffset=0):
         y = y/np.sum(y)*np.size(dz)*binsz/0.1
         plot.pylab.plot(x, y, color='b', linestyle='-')
 
-        # Overlay the model residuals
```

Comment:
I've made these changes in the version that I just committed.  I now see that although 'lim' is not needed here, I've kept it at the new 260, where it is used to set the plot range to +/-5 sigma.  I verified that the plot that the modified code produces for a TS5 data set looks sensible.

I also tried out your new unit test code for xyzPlane_fit, and verified that it passed.